src/test/java/org/cactoos/iterator/SyncIteratorTest.java

```diff
@@ -44,8 +44,9 @@
 public final class SyncIteratorTest {
 
     /**
-     * TODO: There needs to more tests which test for multi-threaded safety
-     * when accessing the iterator.
+     * Puzzle for multi-threaded tests.
```

Comment:
@svendiedrichsen please remove this line, it's unnecessary

Reply:
I have tried but the code check will only accept comments starting with a captital letter. Thus starting the comment with `@todo` will break the build.

Reply:
@svendiedrichsen have you tried the other [formats](https://github.com/yegor256/pdd#how-to-format) available?

Reply:
IMHO only the comment formats are useable here and as the text will extend 80 characters only the multiline comment will do. Which would you suggest otherwise?

Reply:
@svendiedrichsen here's a suggestion: `// @todo #482 add multi-threaded tests`